c/src/lib.rs

```diff
@@ -643,6 +645,13 @@ pub unsafe extern "C" fn foxglove_context_free(context: *const FoxgloveContext)
 #[unsafe(no_mangle)]
 pub extern "C" fn foxglove_internal_register_cpp_wrapper() {
     foxglove::library_version::set_sdk_language("cpp");
+
+    let log_config = std::env::var("FOXGLOVE_LOG_LEVEL")
+        .or_else(|_| std::env::var("FOXGLOVE_LOG_STYLE"))
+        .ok();
+    if log_config.is_some() {
+        foxglove_set_log_level(logging::FoxgloveLogLevel::Debug);
```

Comment:
I think we want info by default.

```suggestion
        foxglove_set_log_level(logging::FoxgloveLogLevel::Info);
```